tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -93,21 +93,9 @@ sub run() {
 
     # create test users
     assert_script_run "useradd -m admin";
-    script_run "passwd admin", 0;    # set user's password
-    type_password "password123";
-    wait_still_screen(1);
-    send_key 'ret';
-    type_password "password123";
-    wait_still_screen(1);
-    send_key 'ret';
-
+    assert_script_run "expect -c 'spawn passwd admin;expect sword:;send password123\\r;expect sword:;send password123\\r;interact'";
```

Comment:
You could just run `echo -e 'admin:password123\nnimda:password123' | chpasswd`

Reply:
changed